.github/workflows/format-check.yml

```diff
@@ -0,0 +1,25 @@
+name: Check Format
+
+on:
+  push:
+    branches: [ 'main' ]
```

Comment:
HI @Taz03 ,
Do we require ticks for branch name in here ?
will this work ?

```suggestion
    branches: [main]
```

Reply:
Yes those are required afaik, or I can expand the list and have `- main`, if u prefer that

Reply:
I believe both will work!

<img width="769" alt="image" src="https://github.com/user-attachments/assets/16ba9ed9-a7ce-482c-bab9-47d3053bb242">

Reply:
so which one do we want?

Reply:
We can go with ` branches: [main]` as @harsha509 suggested. The ticks are not required.